app/views/insured/consumer_roles/_no_match.html.erb

```diff
@@ -1,13 +1,13 @@
 <% if @bs4 %>
-  <div class="alert alert-info d-flex" role="alert">
-    <div>
-      <div class="info-icon icon" alt="info" aria-hidden="true">&nbsp;</div>
+  <div class="alert alert-warning d-flex" role="alert">
+    <div class="pl-2">
+      <div class="warning-icon icon" alt="warning" aria-hidden="true">&nbsp;</div>
     </div>
-    <div class="mt-1 mb-1">
+    <div class="mt-1 pl-0 pr-2 mb-1">
       <% if @consumer_candidate&.match_person.present? && EnrollRegistry.feature_enabled?(:pii_match_success_message) %>
         <%= l10n("insured.consumer_roles.match_info", site_short_name: site_short_name) %>
       <% else %>
-        <%= l10n("insured.consumer_roles.no_match_info", site_short_name: site_short_name) %>
+        <%= l10n("insured.consumer_roles.no_match_info_bs4", site_short_name: site_short_name) %>
```

Comment:
As the copy is being updated for bs4, but doesn't reference bs4, I would change the translation label to something like `no_match_info_2` or `no_match_info_continue_next`

Reply:
Done.